internal/build/docker.go

```diff
@@ -375,6 +374,9 @@ func readDockerOutput(reader io.Reader) (*json.RawMessage, error) {
 			return nil, fmt.Errorf("decoding docker output: %v", err)
 		}
 
+		// TODO(Han): make me smarter! 🤓
+		stdout.Write([]byte(fmt.Sprintf("%+v\n", message)))
```

Comment:
We pass the `io.Writer` all the way through, just to get it here and print out this ugly message. @yuindustries is going to make the output a lot better in the future, but for now we can see what's going on.